.circleci/config.yml

```diff
@@ -346,6 +346,60 @@ jobs:
     - run: node ./scripts/circleci/setup-angular-snapshots.js
     - run: ./scripts/circleci/run-local-browser-tests.sh
 
+
+  # ----------------------------------------------------------------------------
+  # Job that runs all Bazel tests against Ivy with the current Angular version
+  # specified in the project dev dependencies.
+  # ----------------------------------------------------------------------------
+  ivy_test:
+    <<: *job_defaults
+    resource_class: xlarge
+    environment:
+      GCP_DECRYPT_TOKEN: *gcp_decrypt_token
+    steps:
+      - *checkout_code
+      - *restore_cache
+      - *copy_bazel_config
+      - *setup_bazel_remote_execution
+      - *yarn_install
+
+      # Setup Angular ivy snapshots built with ngtsc but locked to a specific tag. We
+      # cannot determine the tag automatically based on the Angular version specified in
+      # the "package.json" because the SHA is not known for the given release. Nor can
+      # we use ngcc to apply the ivy switches because ngcc currently does not handle the
+      # UMD format which is used by Bazel when running tests.
+      - run: node ./scripts/circleci/setup-angular-snapshots.js --tag 8.0.0-rc.0-ivy-aot+bd37622
+      # Disable type checking when building with Ivy. This is necessary because
+      # type checking is not complete yet and can incorrectly break compilation.
```

Comment:
Also link to an issue for this one? I'm guessing there's one for framework and we should also have one on components to make our templates work with the stricter checks

Reply:
Done.